### PR TITLE
Update libicu package information for Kali

### PR DESCRIPTION
**Pull request by kiazhi**
## PR Summary

Fix issue #254 in Dockerfiles due to packages url changes from various repositories.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [x] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: release/community-stable/kali-rolling/docker/Dockerfile [hunk 1]
```diff
@@ -118,4 +118,4 @@ LABEL maintainer="PowerShell Team <powershellteam@hotmail.com>" \
       org.label-schema.docker.cmd.test="docker run ${IMAGE_NAME} pwsh -c Invoke-Pester" \
       org.label-schema.docker.cmd.help="docker run ${IMAGE_NAME} pwsh -c Get-Help"
 
-CMD [ "pwsh" ]
+CMD [ "pwsh" ]
```
Comment (TravisEz13) [inline release/community-stable/kali-rolling/docker/Dockerfile]: Please add a new line to keep the file posix compliant

Comment (TravisEz13) [reply release/community-stable/kali-rolling/docker/Dockerfile]:
```suggestion
CMD [ "pwsh" ]

```

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 2 hours

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 2 hours ago

File: release/community-stable/kali-rolling/docker/Dockerfile (hunk 1 above)
Comment (TravisEz13) [reply release/community-stable/kali-rolling/docker/Dockerfile]: Not blocking

**Pull request merged**